### PR TITLE
HBW-115 Get number of tasks by request to Camunda

### PR DESCRIPTION
**Pull request by mxsmirnoff**

### CODE REVIEW

File: hbw/app/models/hbw/task.rb [hunk 1]
```diff
@@ -29,6 +29,21 @@ def fetch(email, entity_code, entity_class, size = 1000, for_all_users = false)
         end
       end
 
+      def fetch_number_tasks(email, entity_code, entity_class, for_all_users = false)
```
Comment (sugarfree1) [inline hbw/app/models/hbw/task.rb]: number_tasks -> task_count

**Comment by mxsmirnoff** [comment]
Updated

**Comment by mxsmirnoff** [comment]
Updated

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,6 +16,7 @@ v1.7.0 [unreleased]
 -   [#251](https://github.com/latera/homs/pull/251) Update react-select up to 2.1.1.
 -   [#253](https://github.com/latera/homs/pull/253) Drop Activiti support.
 -   [#258](https://github.com/latera/homs/pull/258) Update React & ReactDOM to 16.0.0
+-   [#259](https://github.com/latera/homs/pull/259) Get number of tasks by request to camunda.
```
Comment (sugarfree1) [inline CHANGELOG.md]: Camunda

**Comment by mxsmirnoff** [comment]
Updated

**Pull request merged**